timesketch/frontend/src/assets/main.scss

```diff
@@ -30,7 +30,7 @@
 [data-theme='light'] {
   --background-color: linear-gradient(90deg, rgba(249, 249, 249, 1) 11%, rgba(241, 241, 241, 1) 100%);
   --default-font-color: var(--font-color-dark);
-  --default-link-color: var(--font-color-dark);
+  --default-link-color: #3273dc;
```

Comment:
Where does this color come from? :) It is not consistent with the rest of the UI and this will change all links which is a substantial change.  If you really want blue links here, could you scope them to this component only for now?

Reply:
Ping @tomchop - PTAL

Reply:
Sounds good, I've removed it for now and moved to the local colors

Reply:
👍  The new link color is till in the PR though :) (line 33)